scripts/build-examples.sh

```diff
@@ -5,11 +5,17 @@ dir=${dir-"examples/vg-specs"}
 
 echo "compiling examples to $dir"
 
-mkdir -p $dir
-rm -f $dir/*
-
-for file in examples/specs/*.vl.json; do
-  filename=$(basename "$file")
-  name="${filename%.vl.json}"
-  bin/vl2vg -p $file > $dir/$name.vg.json
-done
+rm -rf $dir
+mkdir $dir
+
+if type parallel >/dev/null 2>&1
+then
+  for file in examples/specs/*.vl.json; do
+    filename=$(basename "$file")
+    name="${filename%.vl.json}"
+    bin/vl2vg -p $file > $dir/$name.vg.json
+  done
+else
+  echo "yeah we use parallel!"
```

Comment:
echo "Using parallel to generate vega specs from examples in parallel"

For the other case, let's also say 


echo "Parallel not found! Sequentially generate vega specs from examples."